lib/urbanopt/geojson.rb

```diff
@@ -33,13 +33,17 @@
 
 module URBANopt
   module GeoJSON
+    # AUTOLOAD is like require but it only loads the module the first time it's used
+    autoload :BuildingCreation, "urbanopt/geojson/building_creation"
```

Comment:
We should *not* use autoload because it gives the custom Ruby interpreter in our CLI troubles.

Reply:
It might be a good idea but it's just too fancy for our CLI :-)